json-schema-store/src/main/java/uk/ac/ebi/biosamples/jsonschemastore/schema/document/SchemaBlock.java

```diff
@@ -0,0 +1,47 @@
+package uk.ac.ebi.biosamples.jsonschemastore.schema.document;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Getter
+@Setter
+@Document(collection = "SchemaBlock")
+public class SchemaBlock {
+  @Id
+  private String id;
+  private String schema;
+  private String title;
+  private String description;
+  private String type;
+  private Object meta;
+  private List<String> required;
+  private Object properties;
```

Comment:
Is it possible to have a map for properties. would there be any benefits having a map?

Reply:
review comment incorporated